### DISCUSSION TITLE
Profile resolver: What are the 5 types of supported mappings?

### DISCUSSION THREAD

**Discussion by galtm** [usnistgov/OSCAL#1114] [Q&A] [answered]
In the [profile resolver spec](https://pages.nist.gov/OSCAL/concepts/processing/profile-resolution/), the Mapping Controls section says, "The Mapping section consists of 5 optional subsections, each covering a particular type of object." The section mentions controls as one of the five types but does not enumerate all five.

**Comment by bradh** [comment]
I was going to suggest that you need to read it in conjunction with the Profile spec (e.g. https://pages.nist.gov/OSCAL/reference/develop/profile/xml-index/) , but I don't even see `mapping` in that spec...

**Reply by galtm** [reply]
@bradh Thanks for your suggestion. I'll keep that in mind in general, and also in case the answer to this particular question shows up there before it appears here.

**Comment by david-waltermire** [comment, marked as answer]
We haven't added mapping yet to the OSCAL models. This is a feature planned for [OSCAL 1.1.0](https://github.com/usnistgov/OSCAL/milestone/9). It relates to issues #538 and #843.
